test/integration/case_study_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class CaseStudyTest < ActionDispatch::IntegrationTest
+  test "translated case study" do
+    setup_and_visit_content_item('translated')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+
+    assert page.has_css?('.available-languages')
+  end
+
+  test "withdrawn case study" do
+    setup_and_visit_content_item('archived')
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+
+    within ('.withdrawal-notice') do
+      assert page.has_text?('This case study was withdrawn'), "is withdrawn"
+      within shared_component_selector("govspeak") do
+        assert_equal @content_item["details"]["withdrawn_notice"]["explanation"], JSON.parse(page.text).fetch("content")
```

Comment:
In test mode `slimmer` doesn't fetch components from `static`. Instead it inserts a placeholder element that contains a JSON representation of the arguments passed to the component partial. eg:


```
<%= render 'govuk_component/title', title: "hello world" %>
```

Is rendered as 
```
<test-govuk-component data-template="govuk_component-title">{ "title": "Hello World" }</test-govuk-component>
```